runbotics/runbotics-orchestrator-ui/src/src-app/Actions/desktop.actions.ts

```diff
@@ -0,0 +1,192 @@
+import { IBpmnAction, Runner } from './types';
+
+// eslint-disable-next-line max-lines-per-function
+const getDesktopActions: () => Record<string, IBpmnAction> = () => ({
+    'desktopAutomation.click': {
+        id: 'desktopAutomation.click',
+        label: 'Click',
+        script: 'desktopAutomation.click',
```

Comment:
you can create DesktopActions enum in runbotics-common and use it for id and script just like it's done for excel actions